test/checks/lists/only-dlitems.js

```diff
@@ -176,6 +176,24 @@ describe('only-dlitems', function() {
 		);
 	});
 
+	it('should return false if the list has dt and dd inside a div group', function() {
+		var checkArgs = checkSetup(
+			'<dl id="target"><div><dt>An item</dt><dd>A list</dd></div></dl>'
+		);
+		assert.isFalse(
+			checks['only-dlitems'].evaluate.apply(checkContext, checkArgs)
+		);
+	});
+
+	it('should return true if the list has dt and dd inside a div group with a role', function() {
```

Comment:
What about a test with other content inside the div?

Reply:
Resolving in a new PR